sources.ui

```diff
@@ -409,7 +409,7 @@ of the appropriate groups to use them.</string>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
-          <string>Any external programs. A string %GDWORD% will be replaced with the query word. If such string is not presented, the word will be fed into standard input.</string>
+          <string>Any external programs. A string %GDWORD% will be replaced with the query word. If such string is not presented, the word will be fed into standard input. A string %GDSEARCH% will be replaced with the text in the searchbar.</string>
```

Comment:
change the label text will make all the translation entry invalid.

maybe need a future translation “Additions and deletions”

Reply:
Do you think it's better to leave the string unchanged? Or add a second QLabel below? Or maybe add translations?

Reply:
It's ok to keep this change